requirements.txt

```diff
@@ -4,4 +4,6 @@ grpcio-tools==1.1.3
 protobuf==3.2.0
 supermutes==0.2.5
 requests==2.14.2
-PyYAML==3.12
\ No newline at end of file
+PyYAML==3.12
```

Comment:
Another PR removed version pinning from our requirements. Please remove yours and resolve the conflict